server/main.py

```diff
@@ -2,13 +2,17 @@
 import json
 import sys
 import traceback
+from os.path import join, dirname, abspath
 
+from quart.cli import load_dotenv
 from quart_cors import cors
 from quart import Quart, jsonify, Response, request
 
 from server.config import Config
 from server.call.operator import Operator
 
+dotenv_path = join(dirname(dirname(abspath(__file__))), '.env')
```

Comment:
Realized when running with `hypercorn` that the .env file is not loaded by default as it seems to be when running with `quart`. So just adding this for consistency between different run types.